packages/v4/src/content/design-guidelines/components/code-block/code-block.md

```diff
@@ -0,0 +1,30 @@
+---
+id: Code block
+section: components
+---
+
+A **code block** is a component that contains two or more lines of ready-only code. The code in a code block can be copied to the clipboard.
+
+## Elements
+Code block has four main components.
+
+<img src="./img/code-block-structure.png" alt="Elements of the code block component" width="441"/>
+
+1. **Code snippet:**  the content that will be copied to the user’s clipboard.
+2. **[Action buttons]:(#Multiple-action-buttons)** completes an action when clicked. The copy button is always present but additional action buttons can be added on an as needed basis.
+3. **Tooltip:** informs users that clicking the button will copy the content to their clipboard, or that the content has successfully been copied.
+4. **[Show more/less (optional)]:(#Show-more/less)** expands and collapses the code snippet.
```

Comment:
You gotta move the semi colon to appear at the end instead of in the middle! Also, don't capitalize the stuff after the #, should be (#multiple..... and (#show-more.....

Reply:
Thanks for catching those! I'll update them.